icu4c/source/i18n/calendar.cpp

```diff
@@ -3695,13 +3695,15 @@ int32_t Calendar::handleGetExtendedYearFromWeekFields(int32_t yearWoy, int32_t w
 
 int32_t Calendar::handleGetMonthLength(int32_t extendedYear, int32_t month) const
 {
-    return handleComputeMonthStart(extendedYear, month+1, true) -
-        handleComputeMonthStart(extendedYear, month, true);
+    const int32_t crtYear = handleComputeMonthStart(extendedYear, month+1, true);
+    const int32_t nextYear = handleComputeMonthStart(extendedYear, month, true);
+    return crtYear - nextYear;
```

Comment:
https://github.com/unicode-org/icu/pull/2767/commits/d5bf628527850f446be20063c11c63b1b9b83695 only fixed handleGetYearLength, it seems like a good idea fix this one too. Though you will want these to be called `Month`, and I think you have current and next swapped.

Reply:
I know :-) Done already.
I was also working to solve the conflicts, but I was running late (missing the bus)
Thanks,
M